README.md

```diff
@@ -1,18 +1,17 @@
-<img src="https://f.cloud.github.com/assets/328359/1930603/3bd6fec6-7eb0-11e3-9945-f41820336d8c.png" alt="Gush logo"  width="90px"/>
+<img src="https://f.cloud.github.com/assets/328359/1930603/3bd6fec6-7eb0-11e3-9945-f41820336d8c.png" alt="Gush logo" width="90px"/>
 
+[![Circle CI](https://circleci.com/gh/gushphp/gush.svg?style=shield&circle-token=:circle-token)](https://circleci.com/gh/gushphp/gush)
 [![Gitter chat](https://badges.gitter.im/gushphp/gush.png)](https://gitter.im/gushphp/gush)
 [![Latest Stable Version](https://poser.pugx.org/gushphp/gush/v/stable.png)](https://packagist.org/packages/gushphp/gush)
 [![Latest Unstable Version](https://poser.pugx.org/gushphp/gush/v/unstable.png)](https://packagist.org/packages/gushphp/gush)
 
 It automates common maintainer and contributor tasks and has
-backend support for GitHub, Enterprise Github, and more! See [here](https://vimeo.com/88283752) and [here](https://vimeo.com/85439368)!
-
-Install by cloning it, building it and symlinking the entry point bin/gush.
+backend support for GitHub, Enterprise GitHub, and more! See [here](https://vimeo.com/88283752) and [here](https://vimeo.com/85439368)!
 
 *Logo explanation is best presented from this passage in Psalms 78*:
 
-> True, he struck the rock, and water gushed out, streams flowed abundantly, but can he also give us bread?
-> Can he supply meat for his people?” When the Lord heard them, he was furious; his fire broke out against
+> "True, he struck the rock, and water gushed out, streams flowed abundantly, but can he also give us bread?
```

Comment:
are you sure the quote starts here? though even if it is not starting here the correction is valid, jsut asking